### ISSUE TITLE
DeprecationWarning: Tapable.plugin & LibSass @import error/warnings

### ISSUE THREAD

**Issue by henjovr** [angular/angular-cli#11202]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Area
```
- [x] devkit
- [ ] schematics
```

### Versions
```
Node v8.9.1
NPM 5.5.1
Windows 10
@angular/cli 6.0.8
```

### Repro steps
Upgraded project from 5.2 to 6, using the upgrade steps. Application is working perfectly, but gives these compile warnings/errors:

`ng build --output-hashing none --prod --progress false`
`ng test --watch=false --browsers PhantomJS --progress false`

### The log given by the failure
```
BUILD log output:

`DEPRECATION WARNING on line 11, column 8 of stdin:
Including .css files with @import is non-standard behaviour which will be removed in future versions of LibSass.
Use a custom importer to maintain this behaviour. Check your implementations documentation on how to create a custom importer.`

TEST log output:
`(node:15240) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead`

```

### Desired functionality
No more errors/warnings.

**Comment by Teamop** [comment]
1. For the LibSass deprecation warning, it belongs to `node-sass`, here are two options:
  - just wait for the upcoming node-sass@4.10 release. https://github.com/sass/node-sass/issues/2362#issuecomment-396239814
  - use `@import url(foo.css);` to download the CSS file, but lose the ability to inline the content to the final CSS file except using a CSS minification step in you build process. https://github.com/sass/node-sass/issues/2362#issuecomment-386259486
2. For the Tapable.plugin deprecation warning, I've already submit a PR #11254

**Comment by HarelM** [comment]
@Teamop Is there any configuration workaround that can be used until this is merged and released? some flag I can add to `ng test` so it won't show this warning?

**Comment by Teamop** [comment]
@HarelM AFAIK, no.

**Comment by alan-agius4** [comment]
Thanks for reporting this issue. This issue is now obsolete due to changes in the recent releases. Please update to the most recent Angular CLI version.

If the problem persists after upgrading, please open a new issue, provide a simple repository reproducing the problem, and describe the difference between the expected and current behavior.

**Comment by HarelM** [comment]
Does this mean this issue is solved in 7.1.2?
